client/view/BrowserStats/browserStats.js

```diff
@@ -1,9 +1,15 @@
 import wikipediaLinks from '../../data/wikipedia-links.js'
 import * as browsersIcons from '../../data/browsers-logos.js'
 
+function formatPercent(percent) {
+  let rounded = percent >= 0.1 ? percent.toFixed(1) : percent.toFixed(0)
```

Comment:
```suggestion
  let rounded = percent >= 0.1 ? percent.toFixed(1) : 0
```

Am I right that `percent.toFixed(0)` will always return `0`? We can make it more clear.

Reply:
Makes sense, thank you! 😄